### ISSUE TITLE
Exception configuring Global Propagators

### ISSUE THREAD

**Issue by mickaelmagniez** [open-telemetry/opentelemetry-java#2181]
**Describe the bug**
Can't configure Global Propagators, lead to `"llegalArgumentException: The OpenTelemetrySdk can only be configured with a TracerSdkProvider"`

**Steps to reproduce**
```scala
OpenTelemetry.setGlobalPropagators(
  DefaultContextPropagators.builder().addTextMapPropagator(HttpTraceContext.getInstance()).build()
)
OpenTelemetrySdk
  .getGlobalTracerManagement()
  .addSpanProcessor(SimpleSpanProcessor.builder(new LoggingSpanExporter()).build())
val tracer = OpenTelemetry.get().getTracer("io.opentelemetry.play")
tracer
```

**What did you see instead?**
`"llegalArgumentException: The OpenTelemetrySdk can only be configured with a TracerSdkProvider"` from the line with `OpenTelemetry.setGlobalPropagators`

**What version and what artifacts are you using?**
0.11.0

**Environment**
Compiler: (e.g., "AdoptOpenJDK 11.0.6")
OS: (e.g., "Ubuntu 20.10")

**PS**
Seems to work in 0.10.0

**Comment by jkwatson** [comment]
Hi @mickaelmagniez This has been fixed on the main branch and will be released with 0.12.0 (planned for this week).

**Comment by jkwatson** [comment]
closing as this has been fixed.
